packages/manager/apps/dedicated/client/app/dedicated/server/interfaces/ola-reset/ola-reset.routing.js

```diff
@@ -0,0 +1,26 @@
+import { redirectTo as pendingTaskRedirectTo } from '../ola-pending-task/ola-pending-task.routing';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('app.dedicated.server.interfaces.ola-reset', {
+    url: '/ola-reset',
+    translations: { value: ['.'], format: 'json' },
```

Comment:
Maybe we can prefer a
```suggestion
    .run(/* @ngTranslationsInject:json ./translations */)
```
annotation